core/config/config.go

```diff
@@ -364,10 +364,11 @@ func migrateAccessTokens(ctx context.Context, db pg.DB, sdb *sinkdb.DB) error {
 		case "network":
 			grant.Policy = "crosscore"
 		}
-		err = sdb.Exec(ctx, store.Save(ctx, &grant))
-		if err != nil {
-			return errors.Wrap(err)
-		}
+		stores = append(stores, store.Save(ctx, &grant))
```

Comment:
I'm not sure how we want to handle that: in the non-atomic migration world, it seems like all but one access token with the same key would've just been overwritten. We can do that overwrite inside `migrateAccessTokens` before we `Exec`, or we can keep the error as-is? Otherwise, if actually want to be able to store multiple `client` or `network` tokens in sinkDB, we may need to change the key that we store them under now